manifests/ocp/01-cr-installation.yaml

```diff
@@ -4,6 +4,10 @@ kind: Installation
 metadata:
   name: default
 spec:
+# Configures Calico networking using the eBPF dataplane. If you want to change the dataplane
+# to iptables, you should set the linuxDataplane parameter below to "Iptables".
```

Comment:
Indentation seems off here

Reply:
Worth adding a comment here saying "You must also change `deployKubeProxy` within `cluster-network-operator.yaml`